libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/TripSessionLocationEngine.kt

```diff
@@ -56,6 +58,7 @@ internal class TripSessionLocationEngine constructor(
 
     @SuppressLint("MissingPermission")
     fun startLocationUpdates(isReplayEnabled: Boolean, onRawLocationUpdate: (Location) -> Unit) {
+        this.isReplayEnabled = isReplayEnabled
```

Comment:
Looks like this has to be moved to after `stopLocationUpdates` call, otherwise `isReplayEnabled` will be reset to `false`.

Reply:
yep added a few tests